model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerPod.java

```diff
@@ -100,7 +154,34 @@ void setLivenessProbe(Integer initialDelay, Integer timeout, Integer period) {
   }
 
   boolean hasV2Fields() {
-    return !this.env.isEmpty();
+    return !this.env.isEmpty()
+        || !this.nodeSelectorMap.isEmpty()
+        || !this.resourceRequirements.getRequests().isEmpty()
+        || !this.resourceRequirements.getLimits().isEmpty()
+        || !this.nodeSelectorMap.isEmpty()
+        || !hasAnySecurityContextConfigured()
+        || !hasAnyPodSecurityContextConfigured();
+  }
+
+  private boolean hasAnySecurityContextConfigured() {
+    return podSecurityContext.isRunAsNonRoot() == null
```

Comment:
This is complicated and will be difficult to remember if they ever add fields to this type.  Let's have the podSecurityContext default to null.

Reply:
Ok, I agree, will change the logic to make it nullable